packages/api-clients/open-api/bffApi.yml

```diff
@@ -16516,6 +16579,19 @@ components:
       required:
         - id
         - name
+    DelegationTenants:
+      type: object
+      additionalProperties: false
+      properties:
+        results:
+          type: array
+          items:
+            $ref: "#/components/schemas/DelegationTenant"
+        pagination:
+          $ref: "#/components/schemas/Pagination"
+      required:
+        - results
+        - pagination
```

Comment:
> According to [the SRS](https://pagopa.atlassian.net/wiki/spaces/PDNDI/pages/1236959344/SRS+Incaricato#Delegato-crea-finalit%C3%A0) the result shall contain a list of objects like the following one, where id is the delegationId - DelegationTenant is a different model, we shall return a list of delegations compacted to display only the three fields below, not a list of tenants
> 
> ```
>     {
>       "id" : "3fa85f64-5717-4562-b3fc-2c963f66afa1",
>       "delegatorId": "3fa85f64-5717-4562-b3fc-2c963f66afa2",
>       "delegatorName": "Delegante Uno" 
>     }
> ```
> 
> (IMO to adhere to the standard use in other routes, I would also change this model to something like `{ id: DelegationId, delegator: DelegationTenant }`), it's what we already do in other parts of the BFF when returning Delegations / CompactDelegations

True, the object I'm returning is different from the one in the SRS, i proposed [here](https://pagopaspa.slack.com/archives/C07U5M9SJ1Y/p1734082954936569?thread_ts=1733905684.079529&cid=C07U5M9SJ1Y) this change since there could be multiple delegationIds for a delegator and the `delegationId` can be retrieved with the next request that lists the eservices.

So I'm returning only the following object
```   
{
   "id": "3fa85f64-5717-4562-b3fc-2c963f66afa2",
   "name": "Delegante Uno" 
}
```

Reply:
It makes sense, I checked the front end and it's just used to populate a dropdown of tenants so it also makes sense semantically that this call returns a list of tenants and not of delegations. Could you add a comment also on the SRS document linking to the Slack thread? So we keep it aligned with the decisions we take while challenging the requirements and implementing